General/general_shacl.ttl

```diff
@@ -0,0 +1,101 @@
+@prefix general: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/general/> .
+@prefix gx: <https://registry.lab.gaia-x.eu/development/api/trusted-shape-registry/v1/shapes/jsonld/trustframework#> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+general:GeneralShape a sh:NodeShape ;
+    sh:property [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node general:DescriptionShape ;
+            sh:order 1 ;
+            sh:path general:description ], 
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node gx:DataResource ;
+            sh:order 2 ;
+            sh:path general:dataResource ],            
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node general:LinksShape ;
+            sh:order 3 ;
+            sh:path general:links ],         
+        [ sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node general:BundleDataShape ;
+            sh:order 4 ;
+            sh:path general:bundleData ];
+    sh:targetClass general:General .
+    
+general:DescriptionShape a sh:NodeShape ;
+    sh:property [ sh:datatype xsd:string ;
+            sh:description "A human readable name of the entity." ;
+            skos:example "3D Model in Grafing" ;
+            sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:name "name" ;
+            sh:order 1 ;
+            sh:path gx:name ],
+        [ sh:datatype xsd:string ;
+            sh:description "A free text description of the entity." ;
+            skos:example "town district, with traffic signs" ;
+            sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:name "description" ;
+            sh:order 2 ;
+            sh:path gx:description ];
+    sh:targetClass general:Description .    
+    
+general:BundleDataShape a sh:NodeShape ;
+    sh:property [ sh:node general:LinkShape ;
+            sh:description "Reference to required assets" ;
+            skos:example "for scenario, link to required hd map" ;
+            sh:name "requiredData" ;
+            sh:order 1 ;
+            sh:path gx:requiredData ],
+        [ sh:node general:LinkShape ;
+            sh:description "Reference to optional related assets" ;
+            skos:example "at hd map, link to optional surface map" ;
+            sh:name "releatedData" ;
+            sh:order 2 ;
+            sh:path gx:releatedData ];
```

Comment:
why gx:relatedData and not general:relatedData?

Reply:
fixed

Reply:
@MircoNierenz There is a small typo -->releatedData  vs relatedData

Reply:
done!